k8s/data-mgt/stage-data-mgt-api.yaml

```diff
@@ -1,20 +1,13 @@
 apiVersion: apps/v1
```

Comment:
Could we also revisit the usage of Helm, just to avoid duplicating config files for staging and production.
Perhaps create an "Insession Task" for this suggestion? 5th sprint? @NoahNsimbe

Reply:
Helm requires some time to setup. It's a full task that might require a sprint. I don't think an "Insession task" is enough to setup helm because it involves changing the entire workflow. This PR is a quick fix

Reply:
Cool, understood. Thanks @NoahNsimbe